examples/vue3-ssr-vite/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vue3-ssr-vite",
+  "private": true,
+  "version": "0.0.0",
+  "scripts": {
+    "dev": "node server",
+    "build": "yarn build:client && yarn build:server",
+    "build:client": "vite build --ssrManifest --outDir dist/client",
+    "build:server": "vite build --ssr src/entry-server.js --outDir dist/server"
+  },
+  "dependencies": {
+    "algoliasearch": "^4.9.3",
+    "qs": "6.5.1",
+    "vue": "^3.1.5",
+    "vue-instantsearch": "https://pkg.csb.dev/algolia/vue-instantsearch/commit/6cc0d9b5/vue-instantsearch",
```

Comment:
it's from CodeSandbox, but let's have it here until we release an alpha.

Reply:
can't it be a file dependency?

Reply:
yeah you're right. it seems to work! (also forgot about that we already use it in other examples)

Reply:
you reverted it, because of the "infinite yarn" bug? did you try out without dependency and directly importing the built files?